payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/CardNumberController.kt

```diff
@@ -264,7 +263,7 @@ internal class DefaultCardNumberController(
      * An error must be emitted if it is visible or not visible.
      **/
     override val error: Flow<FieldError?> =
```

Comment:
Should this (or will this eventually) be a `StateFlow`?

Reply:
That's the goal!